README.md

```diff
@@ -182,12 +182,12 @@ Yeah, totally. I'd love to know how it went.
 [Advice for Short Term Machine Learning Projects](https://rockt.github.io/2018/08/29/msc-advice) by Tim R. is a pretty good starting point! 
 
 **Do you want to share my solution/code to a problem here**? Yeah, sure - why not? 
-Go to [Github issues](https://github.com/NirantK/awesome-project-ideas/issues) in the repository and let me know there. 
+Go to the [GitHub issues](https://github.com/NirantK/awesome-project-ideas/issues) tap in this repository and let me know there. 
```

Comment:
Thanks for the PR! 

Can you please correct tab, to tap? 

So that I can go ahead and merge this?

Reply:
I have opened a new PR, Thanks!